docs/src/_index.md

```diff
@@ -9,24 +9,24 @@ If you're new to Platform.sh, we recommend starting with the **Big Picture**, in
 
 The main requirement of Platform.sh is that you use Git to manage your application code.
 Your project's configuration is driven almost entirely by a small number of YAML files in your Git repository.
-The **Configuration** section covers those in more detail and can serve as both tutorial and quick-reference.
+The **Configuration** section covers those in more detail and can serve as both a tutorial and a quick reference.
 
-Platform.sh supports a number of different programming **Languages** and environments, and it features recommended optimizations for a number of **Featured Frameworks**.
+Platform.sh supports many different programming **Languages** and environments, and it features recommended optimizations for several **Featured Frameworks**.
 
 Finally, you can also get tips for setting up your own **Development** workflow and **Administering** your Platform.sh account.
 
 ## Git Driven Infrastructure
 
-As a Platform as a Service, or PaaS, Platform.sh automatically manages everything your application needs in order to run.  That means you can, and should, view your infrastructure needs as part of your application, and version-control it as part of your application.
+As a Platform as a Service, or PaaS, Platform.sh automatically manages everything your application needs to run.  That means you can, and should, view your infrastructure needs as part of your application, and version-control it as part of your application files.
 
 ### Infrastructure as code
 
 Platform.sh covers not only all of your hosting needs but also most of your DevOps needs. It is a single tool that covers the application life-cycle from development to production and scaling.
 
-You only need to write your code, including a few YAML files that specify your desired infrastructure, commit it to Git, and push.  You don't need to setup anything manually. The web server is already setup and configured, as is any database, search engine, or cache that you specify.
+You only need to write your code, including a few YAML files that specify your desired infrastructure, commit it to Git and push.  You don't need to set up anything manually. The web server is already set up and configured, as is any database, search engine, or cache that you specify.
 
-Every branch you push can be made a fully independent environment&mdash;complete with your application code, a copy of your database, a copy of your search index, a copy of your user files, everything.
-Its automatically generated URL can be sent to stakeholders or to automated CI systems.
+Every branch you push can be made a fully independent environment &mdash; complete with your application code, a copy of your database, a copy of your search index, a copy of your user files, everything.
```

Comment:
:clipboard: 
There shouldn't be spaces around em dashes: https://developers.google.com/style/dashes?hl=en#em-dashes

```suggestion
Every branch you push can be made a fully independent environment&mdash;complete with your application code, a copy of your database, a copy of your search index, a copy of your user files, everything.
```